wordpress-mobile/WordPress-iOS
# Comment: add `canModerate` attribute

Ref: #16685

(Please review #16768 first.)

This adds the `canModerate` attribute to the `Comment` entity. The value is set in `CommentService` when comments are created.

To test:
- Verify the app is using version 126 of the data model.
- Verify the app builds and runs.
- Since the property isn't used by anything yet, you can check the value when `Comment` is created [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/1f883da53773813193d9819a5db68a8f4d53bcd0/WordPress/Classes/Services/CommentService.m#L1151) to make sure it matches your ability to moderate each Comment.

## Regression Notes
1. Potential unintended areas of impact
N/A. Incomplete feature.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. Incomplete feature.

3. What automated tests I added (or what prevented me from doing so)
N/A. Incomplete feature.

PR submission checklist:
